app/src/utils/walletStore.ts

```diff
@@ -114,10 +157,11 @@ export async function initialize({
 		onError
 	}));
 
-	walletNameStore.update((storeValues: WalletNameStore) => ({
-		...storeValues,
-		walletName: getLocalStorage<WalletName>(localStorageKey)
-	}));
+	const walletName = getLocalStorage<WalletName>(localStorageKey);
+
+	if (walletName) {
+		walletNameStore.updateName(walletName);
+	}
```

Comment:
I don't think we need to check if walletName exists.  `getLocalStorage` was designed by Jordan to retrieve the data that it should have.

Reply:
`getLocalStorage` will return `null` if it can't find the item. So if there's no item in local storage, we don't need to update the name store to `null`.

This just prevents calling update name unnecessarily